runtime-common/mimalloc/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=mimalloc
+PKGDES="A high-performance general purpose allocator"
+PKGSEC=libs
+PKGDEP="glibc"
+
+# Default behavior to put libs inside a subdir will cause unnecessary rebuilds on update
```

Comment:
Default behavior (-DMI_INSTALL_TOPLEVEL=OFF) puts libraries inside a sub-directory in /usr/lib, which will cause unnecessary rebuilds on update. Use -DMI_INSTALL_TOPLEVEL=ON to install libraries in /usr/lib.